share/pot/start.sh

```diff
@@ -24,19 +24,17 @@ start-help()
 # $2 the network interface, if created
 start-cleanup()
 {
-	local _pname
+	local _pname _iface
 	_pname=$1
+	_iface=${2}a
 	if [ -z "$_pname" ]; then
 		return
 	fi
-	if [ -n "$2" ] && _is_valid_netif "${2}a" ; then
-		sleep 1 # try to avoid a race condition in the epair driver,
-			# potentially causing a kernel panic, which should
-			# be fixed in FreeBSD 13.1:
-			# https://cgit.freebsd.org/src/commit/?h=stable/13&id=f4aba8c9f0c
-		ifconfig "${2}a" destroy
+	if [ -n "$_iface" ] && _is_valid_netif "$_iface" ; then
+		pot-cmd stop -p "$_pname" -i "$_iface" -s
```

Comment:
This sends in something like`epair15a`, correct? (this is important for looking at the stop command later).